alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/test/resources/engine_config_complete.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is this a duplicate test file or new ? Similar for all other test resource files below ... ?

If duplicate then maybe raise tech debt to rationalise in future ?